site/static/index.html

```diff
@@ -149,9 +149,10 @@
                     type: "datetime",
                 },
                 yAxis: absolute ? {
-                    title: (crate_name.includes("-opt") || crate_name.includes("-check")) ? { text: "" } : {
-                        text: crate_name.startsWith("Summary") ? summaryYAxis : yAxis,
-                    },
+                    // Only the leftmost one ("-check") has its y-axis titled.
```

Comment:
As per the commit message, with this patch applied the graph order from left to right is now check/debug/opt. (Presumably because there is code somewhere that uses alphabetical ordering to choose the graph position, though I haven't confirmed that.)

Reply:
Ah, I missed that -- sorry -- merging and deploying.